pkg/couchdb/database/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@librocco/database",
+	"version": "0.0.1",
+	"main": "./dist/index.js",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "",
+		"preview": "vite preview",
+		"check": "",
+		"check:watch": "",
+		"lint": "prettier --check --plugin-search-dir=. . && eslint .",
+		"format": "prettier --write --plugin-search-dir=. .",
+		"test": "vitest",
+		"test:ci": "CI=true vitest run",
+		"postinstall": ""
```

Comment:
Remove the empty `check`, `check:watch` and `postinstall` scripts

Reply:
Also, you're missing a `typecheck` script, you can copy that over from `pkg/shared`

Reply:
Aaand, for that you'll need `tsconfig.build.json` copied over from `pkg/shared`

Reply:
Does the `tsconfig.build.json` typecheck build files?

Reply:
No, it has emit setup, while, typechecking runs with `--noEmit`. This would conflict if the same file was used, e.g. having `emitDeclarationOnly: true` (in file) and `noEmit: true` (set by cli) would throw an error. Therefore, all build related config is contained in `tsconfig.build.json` while the `tsconfig.json` holds base config (all config safe to use with `--noEmit` flag).

And yeah...`tsconfig.build.json` extends `tsconfig.json` so the base config is reused for build also.